public/jobs.js

```diff
@@ -0,0 +1,356 @@
+async function buildJobsTable(jobsTable, jobsTableHeader, token, message) {
```

Comment:
I take great issue with the organization of this script, but I've seen it in several students' projects so I know it comes from the course. That being said, how would you refactor this code to improve its structure and organization?

Reply:
> Most of the code in this file is given to us as per the snippet provided in the course. The assignment part was only section 6 - Delete. However can you help me understand what do you mean by organization and structure? Do you mean modularizing the functions? It would be nice to learn better coding practices.
> 
> Do you have this course link?:
> https://learn.codethedream.org/orange-alpaca-node-express-class/

I do not know the course curriculum very well because I am new to code the dream, but I will get to know it as time goes by! If my review contradicts the course I think that's ok, though. Programming is full of contradictions and there's always a grey area for _how_ something should be done. Plus, I want to encourage you to be creative and diverge from the exact course content, because that's really good for your learning. One possible way to do that is to identify things we'd like to do differently compared to the course & bring those ideas to you at code review time. Then, you can take a shot at trying to do something _without someone telling you how_, and then I can be there to support you along that more creative path. That's always something I'm trying to foster in these code reviews. 

In this case, though, "modularizing the functions," is definitely along the right lines! Can you expand on that & add more detail for how you would refactor this? Maybe a step by step guide like 

1. Encapsulate all code related to changing color from line __ to line __ into a separate function named `changeColor`
2. Move line __ to __ new function named `jobStatusClickHandler`

You can do as much or as little of this as you want, it's just a little extra thought exercise. **If you're short on time trying to finish up coursework towards the end of this course cycle, feel free to just skip this task -- this is for you not for me, you won't hurt my feelings 😬**